test/bot.test.ts

```diff
@@ -215,6 +218,65 @@ describe('gitpoap-bot', () => {
       ]);
     });
 
+    it('should not create claims for invalid users', async () => {
+      const githubAPIMock = nock('https://api.github.com')
+        // Test that we correctly return a test token
+        .post('/app/installations/29153052/access_tokens')
+        .reply(200, {
+          token: 'test',
+          permissions: {
+            issues: 'write',
+          },
+        })
+
+        // get github login ids
+        .get('/users/test-test-')
+        .reply(200, {
+          message: 'Not Found',
+          documentation_url: 'https://docs.github.com/rest/reference/users#get-a-user',
+        })
+        .get('/users/-')
+        .reply(200, {
+          id: 75544,
+          type: 'User',
```

Comment:
shouldn't all of these return the following? and would it be 200 or 404? 

```
        .reply(200, {
          message: 'Not Found',
          documentation_url: 'https://docs.github.com/rest/reference/users#get-a-user',
        })
```

Reply:
yeah, it returns 200